src/encoded/static/components/typeutils.js

```diff
@@ -296,23 +296,32 @@ export function fileStatusList(session, sessionProperties) {
 
 // Display supersedes/superseded_by lists.
 export const Supersede = ({ context }) => {
-    // Make array of superseded_by accessions
-    let supersededBys = [];
-    if (context.superseded_by && context.superseded_by.length) {
-        supersededBys = context.superseded_by.map(supersededBy => globals.atIdToAccession(supersededBy));
-    }
-
     // Make array of supersedes accessions
     let supersedes = [];
     if (context.supersedes && context.supersedes.length) {
         supersedes = context.supersedes.map(supersede => globals.atIdToAccession(supersede));
     }
 
-    if (supersededBys.length > 0 || supersedes.length > 0) {
+    let supersededByOutput = null;
+    if (context.superseded_by && context.superseded_by.length > 0) {
+        supersededByOutput = (
+            <h4 className="replacement-accessions__superseded-by">
+                <span>Superseded by </span>
+                {context.superseded_by.map((supersededBy, index) => (
+                    <span key={supersededBy}>
+                        {index > 0 ? <span>, </span> : null}
+                        <a href={supersededBy}>{globals.atIdToAccession(supersededBy)}</a>
+                    </span>
+                ))}
+            </h4>
+        );
+    }
+
+    if ((context.superseded_by && context.superseded_by.length > 0) || supersedes.length > 0) {
```

Comment:
This is minor and can be ignored if desired. If you like terse code, you can place this line under the previous if-statement with the if condition of- (supersedes.length > 0)